tests/conftest.py

```diff
@@ -50,7 +50,7 @@ def app(request):
     instance_path = tempfile.mkdtemp()
     app = Flask('testapp', instance_path=instance_path)
     app.config.update(
-        BROKER_URL=os.environ.get('BROKER_URL', 'memory://'),
```

Comment:
Can you just change the default value so one can still test with different brokers?